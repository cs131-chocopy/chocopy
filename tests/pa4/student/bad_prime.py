# Get the n-th prime starting from 2
def get_prime(n:int) -> int:
    candidate:int = 200
    found:int = 0
    while candidate>=0:
        if is_prime(candidate):
		    print(candidate)
        candidate = candidate - 1
    return 0 # Never happens

def is_prime(x:int) -> bool:
    global div
	div=2
    while div%x!=-1 and div < x:
        if x % div == 0:
            return False
        div = div + 1
    return True

# Input parameter
n:int = 15

# Run [1, n]
i:int = 1

div:int = 0

print(get_prime(100))

