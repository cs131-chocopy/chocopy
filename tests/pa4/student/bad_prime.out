199
197
193
191
181
179
173
167
163
157
151
149
139
137
131
127
113
109
107
103
101
97
89
83
79
73
71
67
61
59
53
47
43
41
37
31
29
23
19
17
13
11
7
5
3
2
1
Division by zero
